src/views/NicerDicer.vue

```diff
@@ -1,3 +1,134 @@
 <template>
     <h1>Nicer Dicer</h1>
-</template>
\ No newline at end of file
+    <form action="submit">
+        <button @click.prevent="rollDice" class="roll-button">Roll Dice</button>
+    </form>
+    <div class="counters-container">
+        <div v-for="(counter, index) in counters" :key="index" class="counter-item">
+            <h3>Number {{ index + 1 }}</h3>
+            <p>Total Rolls: {{ counter }}</p>
+            <p>{{ getPercentage(counter) }}% of all rolls</p>
+        </div> 
+    </div>
+    <div class="total-counter-container">
+        <h3>Total Rolls:</h3>
+        <p>{{ totalRolls }}</p>
+        <div class="bar-chart">
+            <div v-for="(counter, index) in counters" :key="index" class="bar" :style="{ height: getBarHeight(counter) + '%'}">
+                <span class="bar-number">Number: {{ index + 1 }} </span>
+                <span class="bar-percentage"> {{ getPercentage(counter) }}% </span>
+            </div>
```

Comment:
Als kleine Spielerei könnte man noch versuchen, den Code-Abschnitt für den "Balken" als eigene Vue-Komponente auszulagern, aber deine Lösung ist ebenfalls vollkommen in Ordnung.